client/state/jetpack-connect/schema.js

```diff
@@ -0,0 +1,13 @@
+export const jetpackConnectSessionsSchema = {
+	type: 'object',
```

Comment:
Since this object uses patternProperties, you'll want to add the prop: `	additionalProperties: false,` so it'll reject keys that don't match the regex. By default any additional properties (that are not listed) are allowed.